src/cookieConsent/cookieConsent.component.tsx

```diff
@@ -111,29 +97,33 @@ const CookieConsent = (
         vertical: 'bottom',
         horizontal: 'left',
       }}
-      ContentProps={{ className: props.classes.root }}
+      ContentProps={{
+        sx: {
+          color: 'primary.contrastText',
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
This isn't needed - the type is inferred correctly without this eslint-disable line